docs/api.adoc

```diff
@@ -7,58 +7,188 @@ v2.0.0, 2016-12
 
 Detailed description of api docs here!
 
-Hosts
+Search
```

Comment:
Let's call out the headers like this:

GET Search
GET Hostvars etc. to be consistent and clear in the menu at the top.